shared/InitialConfig.ts

```diff
@@ -0,0 +1,20 @@
+// Initial configs used by app if network is not available when app is opened for first time. once network is available response is cached and used from then
+// Note: Need to keep this config in sync with actual mimoto response.
```

Comment:
add configuration file name here
In mosip-config, refer inji-default.properties

Reply:
sure. adding

Reply:
added near all properties key